### PR TITLE
Upgrade libs

### PR DESCRIPTION
**Pull request by JakubMatejka**
zejména kvůli https://github.com/jeremydaly/serverless-mysql/releases/tag/v1.3.0, mohlo by vyřešit https://github.com/keboola/developer-portal/issues/292

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,14 +19,12 @@
     "db-migrate-mysql": "^1.1.10",
     "generate-password": "^1.3.0",
     "http-errors": "^1.7.0",
-    "joi": "^12.0.0",
-    "joi-date-extensions": "^1.0.2",
     "jsonwebtoken": "^8.2.0",
     "lodash": "^4.17.4",
     "moment": "^2.22.2",
     "request-promise-lite": "^0.15.0",
     "serverless-mysql": "^1.1.0",
```
Comment (MiroCillik) [inline package.json]: Ale ten `serverless-mysql` si zatial neupdatol? Alebo to bude az v dalsom kroku?

**Comment by JakubMatejka** [comment]
Updatnul, byla to minor verze (https://github.com/keboola/developer-portal/pull/295/files#diff-8ee2343978836a779dc9f8d6b794c3b2R6036)

File: package.json (hunk 1 above)
Comment (JakubMatejka) [reply package.json]: Updatnul, byla to minor verze (https://github.com/keboola/developer-portal/pull/295/files#diff-8ee2343978836a779dc9f8d6b794c3b2R6036)

Comment (JakubMatejka) [reply package.json]: je to v `yarn.lock` z 1.2.0 na 1.3.0 (nevím jestli ti ten link bude fungovat)

Comment (MiroCillik) [reply package.json]: jo, jasne, ja som si nerozklikol ten yarn.lock. OK!

**Pull request merged**